openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -107,6 +111,10 @@ def is_microfrontend_enabled_for_user(self):
             is_course_staff=self.original_user_is_staff
         )
 
+    @property
+    def is_special_exams_enabled(self):
```

Comment:
ditto: that property is not about enabling special exam but make it be available in mfe